bytefield.typ

```diff
@@ -96,15 +112,37 @@
   } else if ( bitheader == none ) {
     range(bits).map(_ => []);
   } else if (type(bitheader) == dictionary) {
+    let numbers = bitheader.at("numbers",default:none) 
+    if msb_first == true {
+      computed_offsets = computed_offsets.map(i => bits - i - 1);
+    }
     range(bits).map(i => [
       #set align(start + bottom)
       #let h_text = bitheader.at(str(i),default: "");
       #style(styles => {
         let size = measure(h_text, styles).width
-        return box(height: size, inset:(left: 50%))[
-          
+        return [
+          #box(height: size,inset:(left: 50%))[
           #if (h_text != "" and bitheader.at("marker", default: auto) != none){ place(bottom, line(end:(0pt, 5pt))) }
           #rotate(bitheader.at("angle", default: -60deg), origin: left, h_text)
+          ]
+          #if (type(numbers) == bool and numbers and h_text != "") {
+              v(-0.5em)
+              align(center, text(bitheader_font_size)[#i])
```

Comment:
just noticed that this is undocumented in the examples.
setting `numbers: true` adds numbers to a bit if it has a specified label